### PR TITLE
LPS-138279 Remove translation when removing a layout

### PR DESCRIPTION
**Pull request by adolfopa**

### CODE REVIEW

**Comment by adolfopa** [comment]
ci:test:relevant

File: modules/apps/translation/translation-service/src/main/java/com/liferay/translation/internal/model/listener/TranslationLayoutModelListener.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import com.liferay.translation.service.TranslationEntryLocalService;
+
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * @author Adolfo Pérez
+ */
+@Component(service = ModelListener.class)
+public class TranslationLayoutModelListener extends BaseModelListener<Layout> {
```
Comment (AliciaGarciaGarcia) [inline modules/apps/translation/translation-service/src/main/java/com/liferay/translation/internal/model/listener/TranslationLayoutModelListener.java:31]: Question:

I feel the  name _TranslationLayoutModelListener_ odd. I my opinion must be _LayoutModelListener_

But if you think that it's ok, I see the code OK

Comment (adolfopa) [reply modules/apps/translation/translation-service/src/main/java/com/liferay/translation/internal/model/listener/TranslationLayoutModelListener.java:31]: We already have a TranslationJournalArticleModelListener in that same package, so I used the same name for consistency.

**Comment by AliciaGarciaGarcia** [comment]
ci:forward

**Comment by austinchiang** [comment]
ci:forward

**Pull request closed without merging**